staging/src/k8s.io/mount-utils/mount_windows.go

```diff
@@ -300,13 +309,13 @@ func (mounter *SafeFormatAndMount) formatAndMountSensitive(source string, target
 		return err
 	}
 	driverPath := volumeIds[0]
-	target = NormalizeWindowsPath(target)
-	output, err := mounter.Exec.Command("cmd", "/c", "mklink", "/D", target, driverPath).CombinedOutput()
-	if err != nil {
-		klog.Errorf("mklink(%s, %s) failed: %v, output: %q", target, driverPath, err, string(output))
-		return err
+
+	// Mount the disk
+	klog.V(4).Infof("Attempting to mount disk %s in %s format at %s", driverPath, fstype, target)
```

Comment:
is log verbosity change intentional here?

Reply:
This change was unintended for this PR. This PR was only meant for the ``IsAbs`` changes. Reverted.